syntax = "proto3";

import "alice/boltalka/generative/service/proto/common.proto";

package NGenerativeBoltalka.Proto;

message TScoringRequest {
    repeated string Context = 1;
    string Prefix = 2;
    string Suffix = 3;
    TPtuneEmbeddings PtuneEmbeddings = 4;
    bool ForceEos = 5;
}
